morph.go

```diff
@@ -462,7 +470,11 @@ func buildHosts(hosts []nix.Host) (resultPath string, err error) {
 		return
 	}
 
-	resultPath, err = nix.BuildMachines(evalMachinesPath, deploymentPath, hosts, nixBuildArg)
+	nixBuildTargets := ""
+	if nixBuildTargetFile != "" {
+		nixBuildTargets = fmt.Sprintf("import %s", nixBuildTargetFile)
```

Comment:
As discussed on IRC, worked ok locally as I was specifying `--target-file=./some/relative/path.nix`, which is a valid Nix path litera. `--target-file=some/relative/path.nix` is not.

@johanot's suggestion is a good solution, because it means passing in an absolute string literal instead of a Nix path literal, which is then implicitly converted to a path internally. Using a Nix path literal would have been possible, but there are some characters are thare (currently) not valid in a Nix path *literal*, notably including `@`.

I have force-pushed a version of the branch with that change incorporated, and rebased on top of current master.